data/json/mapgen/lake_buildings/freshwater_research_station.json

```diff
@@ -42,6 +42,7 @@
       "items": { "#": { "item": "fishing_items", "chance": 10 } },
       "place_vehicles": [
         { "vehicle": "boat_motor_single", "x": 11, "y": 9, "rotation": 270, "chance": 70 },
+        { "vehicle": "lake", "x": 11, "y": 4, "chance": 70 },
```

Comment:
I'm going to recommend: 
{ "vehicle": "lake", "x": 16, "y": 18, "rotation": 270, "chance": 30, "status": 3 },

for placement.  This will put it on the right side of the station and allow the DUKW to spawn reasonably as well.

Reply:
Fixed by moving the spawn points around, the cabin cruiser can spawn there now as well as the DUKW.